index.html

```diff
@@ -25,131 +25,147 @@
+            <div class="text">
+              <div><h4>Multi-Post Stories Gain+Glory</h4></div>
+              <div>
+                <ul class="tag">
+                  <li class="tag1">Ruby on rails</li>
+                  <li class="tag2">css</li>
+                  <li class="tag3">JavScript </li>
+                  <li class="tag4">html</li>
+                </ul>                                                  
+              </div>
+              <div class="action1"><button type="submit" id="2" class="button_f btn">See Project</button></div>  
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="image_placeholder_card"> 
+            <img src="./images/projet4.png" alt="" id="mobile-view-img"> 
+            <div class="text">
+              <div><h4>Multi-Post Stories Gain+Glory</h4></div>
+              <div>
+                <ul class="tag">
+                  <li class="tag1">Ruby on rails</li>
+                  <li class="tag2">css</li>
+                  <li class="tag3">JavScript </li>
+                  <li class="tag4">html</li>
+                </ul>                                                  
+              </div>
+              <div class="action1"><button type="submit" id="3" class="button_f btn">See Project</button></div> 
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="image_placeholder_card">  
+            <img src="./images/projet5.png" alt="" id="mobile-view-img">
+            <div class="text">
+              <div><h4>Multi-Post Stories Gain+Glory</h4></div>
+              <div>
+                <ul class="tag">
+                  <li class="tag1">Ruby on rails</li>
+                  <li class="tag2">css</li>
+                  <li class="tag3">JavScript </li>
+                  <li class="tag4">html</li>
+                </ul>                                                  
+              </div>
+              <div class="action1"><button type="submit" id="4" class="button_f btn">See Project</button></div>  
+            </div>
+          </div>
+        </div>
+        <div class="card">
+          <div class="image_placeholder_card">  
+            <img src="./images/projet6.png" alt="" id="mobile-view-img">
+            <div class="text">
+              <div><h4>Multi-Post Stories Gain+Glory</h4></div>
+              <div>
+                <ul class="tag">
+                  <li class="tag1">Ruby on rails</li>
+                  <li class="tag2">css</li>
+                  <li class="tag3">JavScript </li>
+                  <li class="tag4">html</li>
+                </ul>                                                  
+              </div>
+              <div class="action1"><button type="submit" id="5" class="button_f btn">See Project</button></div>  
+            </div>
+          </div>
+        </div>
+    </section>
```

Comment:
- [ ] As mentioned by the previous reviewer, all the project under my recent work section would be created from array of objects in your js file. Already created code for this section in your index.html file should be removed but created from js file. I would recommend you to follow the suggestions by the previous reviewer to accomplish this. 
![proj-req](https://user-images.githubusercontent.com/26597432/211159962-b0a54988-8551-4ca3-9d4a-8a717850d086.png)